trulens_eval/trulens_eval/tru.py

```diff
@@ -163,6 +168,8 @@ def __init__(
         database_prefix: Optional[str] = None,
         database_args: Optional[Dict[str, Any]] = None,
         database_check_revision: bool = True,
+        snowflake_connection_parameters: Optional[Dict[str, str]] = None,
+        name: Optional[str] = None,
```

Comment:
Is name optional/required?

Reply:
Yeah for backward compatibility we should have it optional for the time being and still allow the old flow.